client/checkout/woopay/email-input-iframe.js

```diff
@@ -530,6 +530,14 @@ export const handleWooPayEmailInput = async (
 		}
 	} );
 
+	if ( woopayEmailInput.value ) {
+		const email = woopayEmailInput.value;
+
+		if ( validateEmail( email ) ) {
+			woopayLocateUser( email, false );
```

Comment:
The locate user function is doing way more than just locate an user. It's adding tracks and opening an iframe so we should rename it to something that actually tells what it is doing. I though of `showIframeToWooPayUserOrShowOptIn` but we're not actually showing the opt-in here. We're just tracking that it shows up even though that happens somewhere else. It might be good to actually move the tracking code to where we actually show the opt-in

Reply:
> We're just tracking that it shows up even though that happens somewhere else.

It happens inside the `dispatchUserExistEvent` when it's `false`.

> It might be good to actually move the tracking code to where we actually show the opt-in

Yes, but we should not track when the response is `rest_invalid_param` [here](https://github.com/Automattic/woocommerce-payments/blob/284bebdd60c3923600b24965048f34f6c0b6737c/client/checkout/woopay/email-input-iframe.js#L405).